### PR TITLE
TST: Test with latest pysat v3.0.1RC

### PR DESCRIPTION
**Pull request by rstoneback**
# Description

Addresses #(issue)

## Type of change

Please delete options that are not relevant.

- Test against pysat v3.0.1 RC

# How Has This Been Tested?

Online Unit Tests

**Test Configuration**:
* GitHub Actions
* Python 3.7, 3.8, 3.9

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

**Comment by rstoneback** [comment]
No Fortran on the Mac machines on GitHub Actions.

**Comment by rstoneback** [comment]
Looks like there is an issue with apex-you on GitHub Actions Windows,
```
ImportError: DLL load failed while importing fortranapex: The specified module could not be found.
```

File: pysat_requirements.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+dask
+netCDF4
```
Comment (rstoneback) [inline pysat_requirements.txt:2]:
```suggestion
netCDF4<1.5.7
```

**Comment by jklenzing** [comment]
There's an issue with numpy 1.17 through 1.19 with apexpy.  My solution to this is to not enforce NEP29 compliance until June 21, 2022, when 1.20 will be the minimum support version.  We will probably not be at a 1.0.0 version until then anyway.

https://numpy.org/neps/nep-0029-deprecation_policy.html

**Comment by rstoneback** [comment]
Ah, ok. Thanks for the heads up.

**Comment by rstoneback** [comment]
I've moved things to numpy < 1.17 but still having issues with the apexpy installation.

File: pysat_requirements.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+dask
+netCDF4<1.5.7
+numpy<1.17
```
Comment (jklenzing) [inline pysat_requirements.txt:3]:
```suggestion
numpy
```
I don't think this is the issue

File: requirements.txt [hunk 3]
```diff
@@ -1,7 +1,7 @@
 aacgmv2
 apexpy
-numpy
-pandas
+numpy<1.17
```
Comment (jklenzing) [inline requirements.txt:3]:
```suggestion
numpy
```
this one too

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -11,8 +11,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9]
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, windows-latest]
+        python-version: [3.8, 3.9]
```
Comment (jklenzing) [inline .github/workflows/main.yml:15]:
```suggestion
        python-version: [3.7, 3.8, 3.9]
```

**Comment by jklenzing** [comment]
Temporarily updated the files to issue a warning and return empty for certain functions if apexpy or OMMBV are not installed.  Neither of these are working on the windows branch, suspect this will have to be fixed up there.  All failing tests are expecting output from one of these packages.

**Comment by jklenzing** [comment]
Closing this branch since we aren't incorporating windows tests for the near future.

**Pull request closed without merging**